### PR TITLE
[TF FE] Add TF Hugging Face model tests into nightly

### PR DESCRIPTION
**Pull request by popovaan**
### Details:
 - Added Hugging Face models to nightly

### Tickets:
 - 132904

### CODE REVIEW

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -755,7 +768,7 @@ jobs:
   Overall_Status:
     name: ci/gha_overall_status
     needs: [Smart_CI, Build, Debian_Packages, Samples, Conformance, ONNX_Runtime, CXX_Unit_Tests, Python_Unit_Tests,
-            CPU_Functional_Tests, TensorFlow_Models_Tests, PyTorch_Models_Tests, NVIDIA_Plugin, Openvino_tokenizers]
+            CPU_Functional_Tests, TensorFlow_Models_Tests_Precommit, PyTorch_Models_Tests, NVIDIA_Plugin, Openvino_tokenizers]
```
Comment (popovaan) [inline .github/workflows/linux.yml:771]: @akashchi do we need to add TensorFlow_Models_Tests_Nightly here?

Comment (rkazants) [reply .github/workflows/linux.yml:771]: let us add it as well

Comment (akladiev) [reply .github/workflows/linux.yml:771]: No need - overall status is for pre-commits, so if you add nightly here, which is launched only for github.event_name == 'schedule', overall_status will never finish for precommit events

File: .github/workflows/linux.yml [hunk 2]
```diff
@@ -501,15 +501,28 @@ jobs:
...
         fromJSON(needs.smart_ci.outputs.affected_components).TFL_FE.test
     needs: [ Build, Smart_CI, Openvino_tokenizers ]
     uses: ./.github/workflows/job_tensorflow_models_tests.yml
     with:
-      runner: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores' }}
-      event: ${{ github.event_name }}
+      runner: 'ubuntu-20.04-8-cores'
+      scope: 'precommit'
+
+  TensorFlow_Models_Tests_Nightly:
```
Comment (ilya-lavrenov) [inline .github/workflows/linux.yml:514]: why do we need a dedicated job for it?

File: .github/workflows/job_tensorflow_models_tests.yml [hunk 3]
```diff
@@ -12,15 +12,15 @@ on:
         type: string
         required: false
         default: '{"image": null}'
-      event:
-        description: 'Event that triggered the workflow. E.g., "schedule" for nightly runs'
+      scope:
```
Comment (rkazants) [inline .github/workflows/job_tensorflow_models_tests.yml:15]:
```suggestion
      model_scope:
```

File: .github/workflows/linux.yml (hunk 2 above)
Comment (popovaan) [reply .github/workflows/linux.yml:514]: We need to separate nightly and precommit models scope. We can do it either by separating jobs or by adding one more parameter to `job_tensorflow_models_tests.yml`, which will define if it is precommit or nightly.

File: .github/workflows/job_tensorflow_models_tests.yml [hunk 4]
```diff
@@ -114,9 +114,8 @@ jobs:
       - name: TensorFlow Models Tests - TF FE
         run: |
           export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
-          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tensorflow/ -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tensorflow/ -m ${inputs.model_scope} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
```
Comment (akashchi) [inline .github/workflows/job_tensorflow_models_tests.yml:117]:
```suggestion
          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tensorflow/ -m ${{ inputs.model_scope }} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
```

File: .github/workflows/job_tensorflow_models_tests.yml [hunk 5]
```diff
@@ -114,9 +114,8 @@ jobs:
       - name: TensorFlow Models Tests - TF FE
         run: |
           export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
-          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tensorflow/ -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tensorflow/ -m ${{ inputs.model_scope }} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
```
Comment (rkazants) [inline .github/workflows/job_tensorflow_models_tests.yml:117]: Please generate report name based on model scope. Use of `TEST-tf_hub_tf_fe.html` always is incorrect

**Review by rkazants — CHANGES_REQUESTED** [review]
Please fix report name

File: .github/workflows/job_tensorflow_models_tests.yml [hunk 6]
```diff
@@ -114,9 +114,8 @@ jobs:
...
         env:
-          TYPE: ${{ inputs.event == 'schedule' && 'nightly' || 'precommit'}}
           TEST_DEVICE: CPU
 
       - name: Upload Test Results
```
Comment (rkazants) [inline .github/workflows/job_tensorflow_models_tests.yml:121]: here we need to use different name for archive based on model scope

**Review by rkazants — APPROVED** [review]

File: .github/workflows/job_tensorflow_models_tests.yml
Comment (akashchi) [inline .github/workflows/job_tensorflow_models_tests.yml:1]: This reusable workflow is also used in the [linux_arm64.yml workflow](https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/linux_arm64.yml#L364). Though, It is disabled, please update the inputs there as well.

**Review by akashchi — CHANGES_REQUESTED** [review]
GHA changes LGTM except for one thing.

File: .github/workflows/job_tensorflow_models_tests.yml
Comment (popovaan) [reply .github/workflows/job_tensorflow_models_tests.yml:1]: Done.

**Review by akashchi — APPROVED** [review]

**Pull request merged**